code/game/jobs/job/marine/squads.dm

```diff
@@ -453,24 +453,53 @@
 	var/list/extra_access = list()
 
 	switch(GET_DEFAULT_ROLE(M.job))
+		if(JOB_SQUAD_MARINE)	//Marines always get a random fireteam
+			var/rng_fireteam = pick("FT1", "FT2", "FT3")
+			assign_fireteam(rng_fireteam, M)
```

Comment:
I'll admit I haven't looked at the entire functionality but if you're going with automatic assignment I would personally fill the fireteams evenly rather than randomly. As funny as it would be to roll the rng to have all the marines in one fireteam.

Reply:
The odds of that are astronomicslly low. Its an easy fix though, but it lookd worde.

Reply:
Very true but I just used the most extreme example to illustrate that - more likely is you'll have things like 4 marines in one fireteam and 1 and 1 in the others which just seems a bit off. But it's of course your choice I just think evening it out would be a better implementation here.